symfit/core/argument.py

```diff
@@ -58,6 +68,7 @@ class Parameter(Argument):
     # Parameter index to be assigned to generated nameless parameters
     _argument_index = 0
     _argument_name = 'par'
+    __slots__ = ['min', 'max', 'fixed']
```

Comment:
Don't forget 'name' like symbols have too.

Reply:
Nope, don't have to. I just define the slots that *my* class needs, and I don't have to care about my parents. 
See https://stackoverflow.com/questions/1816483/python-how-does-inheritance-of-slots-in-subclasses-actually-work

Reply:
Nice, that's good to know.